src/index.js

```diff
@@ -133,6 +139,10 @@ socket.on("change", (path, patch) => {
   store.dispatch(remote.applyRemote(path.replace(path, ""), patch));
 });
 
+socket.on('counter', function (data) {
+  setDoc((doc) => doc.activeUser = data.count);
```

Comment:
I would prefer better namings for these functions and properties
- what's `setDoc` actually doing? 'saveState' or 'setState' maybe?
- what kind of `data` are we talking about here?
- maybe some typing would help along the way